package.json

```
@@ -1,11 +1,17 @@
 {
   "name": "kamrad",
   "version": "1.0.0",
-  "description": "Kuadrant's developer portal",
-  "main": "src/index.ts",
+  "description": "Kuadrant developer portal",
+  "main": "src/js/index.ts",
   "scripts": {
-    "dev": "concurrently \"npm run lint\" \"hugo server\"",
-    "lint": "tsc --watch  --noEmit",
+    "start": "concurrently npm:start:*",
+    "start:hugo": "hugo server -v -D --disableFastRender",
+    "start:webpack": "webpack serve --config webpack.dev.js --hot",
```

Would be good to have tsc running in dev

```suggestion
    "start:webpack": "webpack serve --config webpack.dev.js --hot",
    "start:lint": "tsc --watch --noEmit",
```

---

I don't think it's needed, Webpack will be watching those files and using `tsconfig.json` as reference... practically the same when you run `tsc`.

---

Plus, it will be yet another process running in parallel...

---

I tried it, and it doesn't catch TS errors when developing.
It's true is another process running, but I was thinking in having a _dev mode_, catching errors live

---

I'm not particular fond of having it independently running, when I also have it in my IDE and on a github action... I don't see the need, really... maybe we could tell webpack to notify the errors in dev mode... but I think that having the lint in the IDE + the GH task should be enough.

---

I also have it in my IDE (vscode), but people with a different IDE may have problems making it work. Besides, I like the errors appearing in the terminal in real time.
But anyway, is not important, I can run `tsc --watch` locally.

---

I could make webpack to do it, but IMHO this kind lints are useful in a CI/CD and before a commit if you don't have it in your IDE.